flake.nix

```diff
@@ -103,7 +103,7 @@
           devShells.default = craneLib.devShell {
             inherit LIBCLANG_PATH;
             inputsFrom = [ penumbra ];
-            packages = [ cargo-watch cargo-nextest protobuf ];
+            packages = [ cargo-watch cargo-nextest protobuf cometbft ];
```

Comment:
cc, fellow nix flake user @plaidfinch: i found this was needed to put `cometbft` in the `$PATH` when running `nix develop`, in order to run the summonerd smoke test.

Reply:
oh neat! can i suggest that we perform that flake maintenance in a separate issue, just to keep this narrowly focused upon repairing our failing CI?

Reply:
filed #4369 to keep tabs on upgrading at some point :slightly_smiling_face:

Reply:
thanks! likewise, wanted to let you know in case that affects any possible validator infra. marking this as resolved now that you've seen it :slightly_smiling_face: